models/events.sql

```diff
@@ -184,6 +184,11 @@ select
                     "path": "$.user_properties",
                     "dtype": "object",
                 },
+                {
+                    "name": "event_properties",
+                    "path": "$.event_properties",
+                    "dtype": "json",
```

Comment:
```suggestion
                    "dtype": "object",
```
so that we hit [this](https://github.com/fullstorydev/dbt_fullstory/blob/01ccec85f19ffbe557229e0f1840b66c2adddcd5/macros/json_value.sql#L9) case.

Reply:
I was wondering whether I should be using object or json, thank you!